cisco_duo/assets/logs/cisco-duo.yaml

```
@@ -0,0 +1,185 @@
+id: cisco-duo
+metric_id: cisco-duo
+backend_only: false
+facets:
+- facetType: range
+  groups:
+  - Cisco Duo
+  name: Credits
+  path: cisco.duo.credits
+  source: log
+  type: double
+- groups:
+  - User
+  name: User Email
+  path: usr.email
+  source: log
+- groups:
+  - User
+  name: User Name
+  path: usr.name
+  source: log
+- groups:
+  - Event
+  name: Event Name
+  path: evt.name
+  source: log
+- groups:
+  - Event
+  name: Event Outcome
+  path: evt.outcome
+  source: log
+pipeline:
+  type: pipeline
+  name: Cisco Duo
+  enabled: true
+  filter:
+    query: "source:cisco-duo"
+  processors:
+    - type: date-remapper
+      name: Define `isotimestamp`, `ts` as the official date of the log
+      enabled: true
+      sources:
+        - isotimestamp
+        - ts
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
+    - type: pipeline
+      name: Cisco Duo Authentication
+      enabled: true
+      filter:
+        query: "service:authentication"
+      processors:
+        - type: geo-ip-parser
+          name: GeoIP Parser for `access_device.ip`
+          enabled: true
+          sources:
+            - access_device.ip
+          target: access_device.geoip
+          ip_processing_behavior: do-nothing
+        - type: geo-ip-parser
+          name: GeoIP Parser for `auth_device.ip`
+          enabled: true
+          sources:
+            - auth_device.ip
+          target: auth_device.geoip
```

Same comment as before. Does this correspond with the client ? If that's the case maybe it could be remapped to : https://docs.datadoghq.com/standard-attributes/?search=network.client.geoip

```suggestion
          target: network.client.geoip
```

---

The IP address of the authentication device which is used to approve or deny duo request, so this won't correspont with a client.

---

Thanks for the clarification.